src/index.ts

```diff
@@ -3,4 +3,14 @@ import http from 'http';
 
 const server: Server = new Server();
 
-const app: http.Server = server.start();
+let app: http.Server;
+
+server
+  .start()
+  .then((application: http.Server) => {
+    app = application;
```

Comment:
I'm not quite clear what the assignment of this value is for. Are we planning on doing something with this value later?

Reply:
No we're not currently, but I'll likely use it to implement graceful shutdown later. So it needs to be defined at the module scope.

Reply:
Alright, got it.